app/routes/explorer.js

```diff
@@ -20,21 +20,27 @@ export default class ExplorerRoute extends Route {
 
   async model({ id }) { // eslint-disable-line
     let selectionResponse = null;
-    let profileResponse = null;
+    let acsSurveyResponse = null;
```

Comment:
For when resolving conflicts: Move these changes to the new `utils/fetch-explorer-model.js` file